index.html

```diff
@@ -877,8 +878,8 @@ <h2>Definition of Roles</h2>
 			<rdef>alert</rdef>
 			<div class="role-description">
 				<p>A type of <a>live region</a> with important, and usually time-sensitive, information. See related <rref>alertdialog</rref> and <rref>status</rref>.</p>
-				<p>Alerts are used to convey messages that might be immediately important to users. In the case of audio warnings, alerts provide an accessible alternative for hearing-impaired users. The <code>alert</code> <a>role</a> is applied to the element containing the alert message. An <code>alert</code> is a specialized form of the <rref>status</rref> role, which is processed as an atomic <a>live region</a>.</p>
-				<p>Alerts are assertive live regions, which means they cause immediate notification for assistive technology users. If the operating system allows, the <a>user agent</a> SHOULD fire a system alert <a>event</a> through the accessibility API when the <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> alert is created.</p>
+				<p>Alerts are used to convey messages that will be immediately important to users. In the case of audio warnings, visibly displayed alerts provide an accessible alternative to audible alerts for Deaf or hard-of-hearing users. Likewise, alerts can provide an accessible alternative to the visible alerts for Blind, DeafBlind, or low vision uses, and others with certain developmental disabilities. The <code>alert</code> <a>role</a> is applied to the element containing the alert message.</p>
```

Comment:
I would probably not capitalize deaf, blind, or deaf-blind. Other WAI pages seem to use "deaf-blind". I think Deaf is capitalized when it refers to someone who identifies with the Deaf community.

Reply:
The Deaf community is pretty adamant about the capital D. I will change the others.

Reply:
```suggestion
				<p>Alerts are used to convey messages that will be immediately important to users. In the case of audio warnings, visibly displayed alerts provide an accessible alternative to audible alerts for Deaf or hard-of-hearing users. Likewise, alerts can provide an accessible alternative to the visible alerts for blind, deaf-blind, or low-vision users, and others with certain developmental disabilities. The <code>alert</code> <a>role</a> is applied to the element containing the alert message.</p>
```